draft-tschofenig-rats-psa-token.md

```diff
@@ -426,14 +427,14 @@ Each entry in the Software Components list describes one software component
 using the attributes described in the following subsections.  Unless explicitly
 stated, the presence of an attribute is OPTIONAL.
 
-Note that, as described in {{?I-D.ietf-rats-architecture}}, a relying party
-will typically see the result of the verification process from the Verifier in
-form of an attestation result, rather than the "naked" PSA token from the
-attesting endpoint.  Therefore, a relying party is not expected to understand
-the Software Components claim.  Instead, it is for the Verifier to check this
-claim against the available endorsements and provide an answer in form of an
-"high level" attestation result, which may or may not include the original
-Software Components claim.
+Note that, as described in {{?RFC9334}}, a relying party will typically see the
+result of the verification process from the Verifier in form of an attestation
+result, rather than the "naked" PSA token from the attesting endpoint.
```

Comment:
Is the word naked really needed here?

Reply:
even better :-)

Reply:
removed in [f042562](https://github.com/thomas-fossati/draft-psa-token/pull/73/commits/f042562bc7a8a1ade7a242015578ada86d6611c4)